cmd/certificates/main.go

```diff
@@ -63,7 +61,17 @@ func cmdRun(cmd *cobra.Command, args []string) error {
 
 	identity, err := runCfg.Identity.Load()
 	if err != nil {
-		zap.S().Fatal(err)
+		return err
+	}
+
+	signer, err := runCfg.Signer.Load()
+	if err != nil {
+		return err
+	}
+
+	authorizationDB, err := authorizations.NewDBFromCfg(runCfg.Authorizations)
+	if err != nil {
+		return errs.New("Error opening authorizations database: %+v", err)
```

Comment:
errors should start lower case

Reply:
Will change, but this was borrowed from the revocation db error messages that we have in all the commands, so those have the same issue.